### PR TITLE
Update Get-CsAutoAttendant.md

### PR DESCRIPTION
**Pull request by CLYVR**
Adding a note about a known issues with the -IncludeStatus

### CODE REVIEW

**Comment by CLYVR** [comment]
@get-itips @dariomws @tseward @tiburd - Please and thanks!

**Comment by tseward** [comment]
@CLYVR is this issue just for v4.0.0 or v4.x+?

**Comment by CLYVR** [comment]
@tseward Starts with 4.0.0 so probably better if that says v4.x+ as you've noted.

**Comment by CLYVR** [comment]
@tseward Updated to say "starting with v4.0.0"

**Review by tseward — APPROVED** [review]

File: skype/skype-ps/skype/Get-CsAutoAttendant.md [hunk 1]
```diff
@@ -187,6 +187,11 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+> [!NOTE]
+> Expect slow response times when using this parameter with Teams Powershell module starting with v4.0.0.  Please use the **-Skip** and **-First** parameters to limit the result size instead.  This issue is expected to be resolved within the next few months.
+
+
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsAutoAttendant.md:193]:
```suggestion
**Note**: Expect slow response times when using this parameter with Teams PowerShell module starting with version 4.0.0.  Use the **-Skip** and **-First** parameters to limit the result size instead.  This issue is expected to be resolved within the next few months.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @CLYVR, kindly commit the suggested changes. Many thanks.

**Comment by tiburd** [comment]
@CLYVR 
Hi, Colin,
In case you don't know, I won't be able to merge this PR until after the error in the changed file is fixed:

> image

If you need assistance, you can post a question in the PubOps [Ask an Admin Teams channel](https://aka.ms/askanadmin).

Thanks!

**Review by tseward — APPROVED** [review]

**Review by dariomws — APPROVED** [review]

**Comment by yogkumgit** [comment]
@get-itips

**Review by get-itips — APPROVED** [review]
approving, but, what if we use _releases_ instead of _months_? @CLYVR

**Comment by CLYVR** [comment]
@get-itips Updated to say "This issue is expected to be resolved in a future release."

**Comment by get-itips** [comment]
> @get-itips Updated to say "This issue is expected to be resolved in a future release."

Awesome, thanks.

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**